lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -875,6 +875,28 @@ function dosomething_reportback_get_reportback_total_by_nid($nid, $type = 'quant
 }
 
 /**
+ * Returns the total number of reportbacks with the override added to it.
+ *
+ * @param int $nid
+*   A node nid.
+ *
+ * @return int $total
+ *  Sum of all reportback quantities plus the override.
+ */
+ function dosomething_reportback_get_reportback_total_plus_override($nid) {
+   // Select all reportbacks for given $nid.
+   $query = db_select('dosomething_reportback', 'rb')->condition('nid', $nid);
+   // Get quantity of all reportbacks
+   $query->fields('rb', array('quantity'));
+   $query->addExpression('SUM(quantity)', 'quantity');
+   $result = $query->execute()->fetchAll();
+   $quantity = $result[0]->quantity;
```

Comment:
you could just call the other function `dosomething_reportback_get_reportback_total_by_nid` instead of copying the query here.